### PR TITLE
Document package showlinked command with OpenAPI

### PR DESCRIPTION
**Pull request by eduardoj**
For reviewers, access directly to the documented endpoint in the review app through this [link](https://obs-reviewlab.opensuse.org/eduardoj-apidocs_newpackage_cmd_showlinked/apidocs-new/index#/Sources%20-%20Packages/post_source__project_name___package_name__cmd_showlinked).

### CODE REVIEW

**Review by danidoni — APPROVED** [review]

File: src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_showlinked.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+post:
+  summary: List all package instances linking to this one.
```
Comment (hennevogel) [inline src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_showlinked.yaml:2]:
```suggestion
  summary: List all packages linking to this one.
```

Comment (eduardoj) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_showlinked.yaml]: Applied.

File: src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_showlinked.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  description: List all package instances linking to this one. This command doesn't perform any action.
```
[the remark this answers is not in the document]
Comment (eduardoj) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_showlinked.yaml]: Applied.

I preserved the sentence 'This command doesn't perform any action.'. This clarifies that, despite being a POST endpoint with a `cmd` (command) parameter, no action will be perfomed. We already had [this case](https://github.com/openSUSE/open-build-service/blob/cef426c08dae9bc6ce5de05d2317405c13d01720/src/api/public/apidocs-new/paths/source_project_name_cmd_showlinked.yaml#L4) before.

[the remark this answers is not in the document]
Comment (hennevogel) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_showlinked.yaml]: I would add more clarification to this then. Like:

Despite being using the method POST, this action doesn't alter any data.

or something.

**Review by hennevogel — APPROVED** [review]

**Comment by danidoni** [comment]
Codecov is stalled, merging

**Pull request merged**